### PR TITLE
Remove the windows-sys dependency

### PR DESCRIPTION
**Pull request by gabrielesvelto**
Resolves: #58

### CODE REVIEW

**Comment by gabrielesvelto** [comment]
The bindings are lifted from windows-sys, save for the structures used in minidump callbacks which are replaced with dummies (we don't use them and they use a zillion different types). This isn't great because I have to make the bindings public in order to use them from the tests; I'm not sure if there' s a way to expose them just to the test but not the the crate's users.

File: build.rs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+fn main() {
+    #![cfg(all(target_os = "windows"))]
+    println!("cargo:rustc-link-lib=dylib=dbghelp");
+}
```
Comment (Jake-Shadle) [inline build.rs:4]:
```suggestion
    if std::env::var("CARGO_CFG_TARGET_OS").as_deref() == Ok("windows") {
        println!("cargo:rustc-link-lib=dylib=dbghelp");
    }
}
```

**Review by Jake-Shadle — CHANGES_REQUESTED** [review]
Just one minor change needed to ensure cross compilation works. Not sure why the build is freaking out about build.rs though

**Comment by Jake-Shadle** [comment]
Oh nvm, it's because it's an outer attribute so applies to the main function so my change would also fix the build issue.

**Review by Jake-Shadle — APPROVED** [review]
:fire:

**Pull request merged**